src/quo/components/text_combinations/page_top/style.cljs

```diff
@@ -6,10 +6,11 @@
    :padding-horizontal 20
    :row-gap            8})
 
-(def header
+(defn header
+  [title-size]
   {:flex-direction  :row
    :justify-content :space-between
-   :height          32})
+   :height          (if title-size :auto 32)})
```

Comment:
Do we know why `:heading-1` would always be height `32`, is that the max-height or something?

Reply:
No idea, just wanted to add possibility to change it from parent without breaking any other usages.